README.md

```diff
@@ -122,6 +122,7 @@ Here are the public commands available.
 - `nxPackage`: Audit (for now) whether a package is installed or not in a system (currently supports apt only).
 - `nxFileLine`: Ensure an exact line is present/absent in a file, and remediate by appending, inserting, deleting as needed.
 - `nxFileContentReplace`: Replace the content in a file if a pattern is found.
+- `nxScript`: Simple resource for executing scripts.
```

Comment:
nit: Simple resource for executing scripts **in PowerShell Core/PowerShell 7**
(Just want customers to know that we aren't running this as a bash or python or PS5 script)